ui/src/app/components/error-handler.tsx

```diff
@@ -98,12 +112,20 @@ export const ErrorHandler = withGlobalError()(class extends React.Component<Prop
 
     return globalError !== undefined && <React.Fragment>
       {globalError.statusCode === 500 && <div style={styles.errorHandler}>
-        Server Error (500)
-        <ClrIcon shape='times' style={styles.iconStyles} onClick={() => this.closeError()} />
-        </div>}
+        <FlexRow style={styles.errorContent}>
+          <FlexColumn>
+          Server Error (500)
+          <div>
+            Please click to copy unique code for support ticket
+            {globalError.errorUniqueId && <CopyToClipboard text={globalError.errorUniqueId}>
```

Comment:
Yeah, I think you'll need to make this conditioned on the `errorUniqueId` being non-null and non-empty.

Reply:
Sure! Will do.